modules/javafx.graphics/src/main/java/com/sun/marlin/DualPivotQuicksort20191112Ext.java

```diff
@@ -767,9 +777,11 @@ private static boolean tryMergeRuns(DPQSSorterContext sorter, int[] a, int[] b,
     /**
      * Merges the specified runs.
      *
-     * @param srcA the source array
-     * @param dstA the temporary buffer used in merging
+     * @param srcA the source array for the array to be sorted (a)
+     * @param dstA the temporary buffer used in merging (a)
+     * @param srcB the source array for the secondary array to be ordered (b)
      * @param offset the start index in the source, inclusive
+     * @param dstB the temporary buffer used in merging (b)
```

Comment:
Minor: this should probably be moved before `offset`

Reply:
Sure, will fix this line before integration